_posts/2022-04-28-introducing-uswds-3-0.md

```diff
@@ -19,24 +19,24 @@ For most USWDS 2 projects, migrating to USWDS 3.0 should only require changing
+If you have any questions or feedback let us know at the [USWDS Public Slack](https://gsa-tts.slack.com/archives/uswds-public), [GitHub Discussions](https://github.com/uswds/uswds/discussions), or [uswds@gsa.gov](mailto:uswds@gsa.gov). 
 
-If you aren't a member of the USWDS Public Slack, [sign up here](https://chat.18f.gov/) by "Joining a TTS Chat Room" —then select "U.S. Web Design System" as your chat topic.
+If you aren't a member of the USWDS Public Slack, [sign up here](https://chat.18f.gov/) by "Joining a TTS Chat Room" — then select "U.S. Web Design System" as your chat topic.
 
 ## What's new in USWDS 3.0?
 USWDS 3.0 takes a modular, component-centered approach to the design system. We've rebuilt our codebase with a component focus so teams can more effectively integrate the design system incrementally and use only the USWDS components needed in your project.
 
 ### Update to modern Sass syntax 
 
-USWDS 3.0 stylesheets use the most modern version of the Sass Module syntax. The Sass language that we use to write our stylesheets updated its syntax in 2020 and the old syntax (using `@import`) is now deprecated. Sass will no longer support the old syntax starting in October 2022. We want teams to use source code with reliable support throughout the life of their project, and USDS 3.0 allows teams to migrate to Sass Modules and maintain support.
+USWDS 3.0 stylesheets use the most modern version of the Sass module syntax. The Sass language that we use to write our stylesheets updated its syntax in 2020 and the old syntax (using `@import`) is now deprecated. Sass will no longer support the old syntax starting in October 2022. We want teams to use source code with reliable support throughout the life of their project, and USWDS 3.0 allows teams to migrate to Sass modules and maintain support.
 
-[Read more about the Sass Module syntax](https://sass-lang.com/) → 
+[Read more about the Sass module syntax](https://sass-lang.com/)
 
 ### Improve performance and reduce the size of your project CSS 
 
-Unbundle the design system with USWDS 3.0. Using USWDS 3.0 with Sass Module syntax allows teams to unbundle their implementations of USWDS and use only the components and code that they need on their project. Depending on your project, this could mean significant reduction of CSS code you ship. This means faster load times, better scores in performance evaluation tools, and a better developer experience. 
+Unbundle the design system with USWDS 3.0. Using USWDS 3.0 with Sass module syntax allows teams to unbundle their implementations of USWDS and use only the components and code that they need on their project. Depending on your project, this could mean significant reduction of CSS code you ship. This means faster load times, better scores in performance evaluation tools, and a better developer experience. 
 
-[Read more about USWDS Packages](#) → 
+[Read more about USWDS Packages](/components/packages/) →
```

Comment:
```suggestion
[Read more about USWDS Packages]({{ site.baseurl }}/components/packages/) →
```

Reply:
Updated in [769f8c3](https://github.com/uswds/uswds-site/pull/1561/commits/769f8c3ac529340c57d075dcaf71a70af01e120c)